libs/wingsdk/src/cloud/counter.ts

```diff
@@ -68,6 +68,13 @@ export interface ICounterClient {
    * @inflight
    */
   inc(amount?: number): Promise<number>;
+
+  /**
+   * Get the current value of the counter
```

Comment:
We should add some kind of warning about the potential for race conditions:
```suggestion
   * Get the current value of the counter.
   * Using this API may introduce race conditions since the value can change between
   * the time it is read and the time it is used in your code.
```